tests/e2e/README.md

```diff
@@ -85,25 +85,28 @@ structures and calls teardown() while framework cleanup. The cleanable register
 
 E2E tests have multiple options available while running them as follows:
 
-* `--skip_cleanup` - to skip cleanup steps
-* `--namespace <namespace>` : If you don't specify `namespace`, a random namespace is generated for each test.
-* `--verbose <debug level noise from proxies>`
-* `--istioctl <local istioctl path>`: Use local istioctl binary (i.e. `${GOPATH}/out/linux_amd64/release/istioctl`).
-* `--istioctl_url <remote istioctl url>`: If local path is not defined, download istioctl from a remote location.
-* `--use_local_cluster`: If running on minikube, this should be set to true.
-* `--auth_enable` - if you want to include auth
-* `--cluster_wide` - if you want to run the cluster wide installation and tests
-* `--use_automatic_injection` - if you want to do transparent sidecar injection
-* `--use_galley_config_validator` - if you want to enable automatic configuration validation
-* `--mixer_hub <mixer image hub>`
-* `--mixer_tag <mixer image tag>`
-* `--pilot_hub <pilot image hub>`
-* `--pilot_tag <pilot image tag>`
-* `--proxy_hub <proxy image hub>`
-* `--proxy_tag <proxy image tag>`
-* `--ca_hub <CA image hub>`
-* `--ca_tag <CA image tag>`
-* `--galley_hub <galley image hub>`
-* `--galley_tag <galley image tag>`
-
-
+* `--skip_setup` - Skip namespace creation and istio cluster setup (default: false)
+* `--skip_cleanup` - Skip the cleanup steps (default: false)
+* `--namespace <namespace>` - If you don't specify `namespace`, a random namespace is generated for each test.
+* `--use_local_cluster` - If true will use NodePort for the services. If running on minikube, this should be set to true. (default: false)
```

Comment:
If true the tests will run within the cluster.

I don't think 'use NodePort for the services' improve the readability.

Reply:
Keep minikube part?

Reply:
Yes. that's important.